uktrade/data-hub-frontend
# Feature/uk postcode

## Description of change

Added the _UK postcode_ filter to _companies_ page.

## Test instructions

Note, that the feature depends on the uktrade/data-hub-api#2441 PR.

1. Go to [`/companies`](https://data-hub-frontend-dev-pr-2358.herokuapp.com/companies?sortby=modified_on%3Adesc&archived=false)
2. In the sidebar, right underneath the _Country_ filter you should see the _Uk postcode_ filter.
3. The filter should conform to the specification
4. Enter a postcode (even an incomplete one) or multiple coma-separated list of postcodes to the filter
5. Blur the input field, or hit enter
6. The search results should now only show companies with the postcodes you entered
 
## Screenshots
### Before
<img width="302" alt="Screen Shot 2020-01-10 at 2 34 17 PM" src="https://user-images.githubusercontent.com/2333157/72160534-63d21c00-33b6-11ea-850e-3cb3bb19474b.png">

### After 
<img width="302" alt="Screen Shot 2020-01-10 at 2 34 03 PM" src="https://user-images.githubusercontent.com/2333157/72160599-7b110980-33b6-11ea-96e2-e8507c744562.png">

## Checklist
